plugins/woocommerce-admin/client/marketplace/stylesheets/_variables.scss

```diff
@@ -5,12 +5,14 @@ $small-gap: $grid-unit-10;   // 8px
 $medium-gap: $grid-unit-20;  // 16px
 $large-gap: $grid-unit-30;   // 24px
 $xlarge-gap: $grid-unit-40;  // 32px
+$huge-gap: $grid-unit-60;    // 48px
```

Comment:
Lol, I can see us graduating to `$ultra-massive-gap` at some point. Would it make sense to rename `$xlarge-gap` to `$large-gap` and making this one `$xlarge-gap`? I'm even wondering if we really need to create synonyms for `$grid-unit-20` etc – perhaps it would make sense to use them directly.

Reply:
Yes, it doesn't make sense to have `$huge-gap`. I removed as it won't be used often in our codebase. Thank you!